addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js

```diff
@@ -178,7 +178,8 @@ odoo.define('point_of_sale.PaymentScreen', function (require) {
                     syncedOrderBackendIds = await this.env.pos.push_single_order(this.currentOrder);
                 }
             } catch (error) {
-                this.error = true;
+				if (error.code == 700)
```

Comment:
Can you fix the indentation? Use spaces to be sure.

Reply:
Hi,
I used the same indentation, and as u know js doesn't care about indentation

Reply:
I know js doesn't care about the indentation. But I bet our lead won't approve this.
Besides, it's not difficult to do.
1. Make the change locally.
2. run `git add .` then `git commit --amend`.
3. force push using `git push <remote-origin> <branch> --force` (make sure this command replaces the commits in this branch).

Reply:
OK, Thanks 
Done

Reply:
Can you align the new `if` to the `if` following it? Also, the commit message is changed, it should be in format:
`[FIX] point_of_sale: <title-of-the-commit>`.

I'm sorry for being persistent. We, internal developers, are following the guidelines, so we would like you to also follow it.

Reply:
of course I will follow the guidelines, I appreciate your help

I think now is fine